server/protocols/webadmin/webadmin-cassandra/src/main/java/org/apache/james/webadmin/routes/CassandraMigrationRoutes.java

```diff
@@ -34,7 +34,6 @@
 
 public class CassandraMigrationRoutes implements Routes {
 
-
```

Comment:
do we need to warn admin team about this commit?

Reply:
They are not supposed to know about Cassandra migration management (yet)

Reply:
I mean changing default output fro html+json to plain json

Reply:
it's probably what they expect and didn't noticed the bug yet. But, yes, we'll tell them.